server/bin/pbench-index.py

```diff
@@ -114,7 +119,7 @@ def main(options, name):
             " use --config <file> on the command line",
             file=sys.stderr,
         )
-        return 2
+        return BAD_CFG
```

Comment:
We need to be careful here because we are now changing the return value.  Let's make sure we don't change any return values as part of this change.

Reply:
ok